user-guide/Getting_started/DaaS/Creating_a_DaaS_system_FAQ.md

```diff
@@ -19,6 +19,13 @@ The DataMiner credit balance of an organization is displayed in the [Admin app](
 
 Contact [your Account Manager](https://community.dataminer.services/get-in-touch/sales-team/).
 
+### When does a DaaS system expire?
+
+A DataMiner Express DaaS system will be scheduled to expire as soon as the organization does not have sufficient DataMiner credits to extend its lifetime.
+
+> [!NOTE]
+> If an organization has multiple DaaS systems, they will be extended in chronological order. A DaaS system can be marked for expiration while the organization still has a sufficient amount of DataMiner credits to extend one or more other DaaS systems, but wont for this particular system.
```

Comment:
Is this a correct interpretation of what you mean?
```suggestion
> If an organization has multiple DaaS systems, they will be extended in chronological order: the system that was created first will be extended first. If your organization does not have enough DataMiner credits left to extend all of its DaaS systems, the system or systems that were created last will be scheduled to expire.
```

Reply:
It's sort of correct but to me it could be confusing to use 'created first'.
Eg: Create system A on day 0, (A is extended on after day 7), create system B on day 8, on day 9 you could receive a notification that system A will expire because the organization doesn't has any credits left but it was the first one created

Reply:
Ok, how about this? I'm trying to find some way to make this clear without creating possible confusion:
```suggestion
> If an organization has multiple DaaS systems, they are extended in the order they were created. When we detect that your organization will not have enough credits left to extend one of your DaaS systems, that DaaS system will be scheduled to expire. However, at this point, you may still have a number of other credits left that will be considered reserved for your other DaaS systems.
```

Reply:
maybe 'If an organization' -> 'when an organization' ?
they are extended in the order they were created. -> I would still go with 'they are extended in chronological order'
'When we detect that your organization' -> 'When your organization'
number of other credits left -> number of DataMiner credits left

Reply:
Could there be another alternative to "chronological order"? My worry is that this is rather vague and will cause more questions as to what exactly we mean with this.

Reply:
Not something I can come up with rn 😔

Reply:
Next attempt:
```suggestion
> When an organization has multiple DaaS systems, they will be extended in chronological order. When your organization does not have enough credits left to extend one of your DaaS systems, that DaaS system will be scheduled to expire. However, at this point, you may still have a number of DataMiner credits left that will be considered reserved for your other DaaS systems.
```